docs/docusaurus/docs/terms/datasource.md

```diff
@@ -3,63 +3,42 @@ title: Datasource
 id: datasource
 hoverText: Provides a standard API for accessing and interacting with data from a wide variety of source systems.
 ---
-
-import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
-import ConnectHeader from '/docs/images/universal_map/_um_connect_header.mdx';
-import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
-import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
 import TechnicalTag from '../term_tags/_tag.mdx';
 
 import InProgress from '/docs/components/warnings/_in_progress.md'
```

Comment:
I think we may be able to remove this now.

Reply:
@Rachel-Reverie - done!